CIP-XXXX/README.MD

```diff
@@ -0,0 +1,146 @@
+
+### Efficient Fund Tracking with Optional Privacy
+By linking all stake keys to the first payment key, this proposal introduces a transparent and simple mechanism for multi-delegation, greatly simplifying the fund tracking process. This association not only simplifies the delegation and deregistration processes but also facilitates easier tracking of funds and delegations. Besides, this method does not preclude the use of the multi-account approach for users desiring enhanced privacy. Stakeholders can still derive multiple accounts, each potentially capable of multi-delegation, granting them granular control while retaining the option for increased privacy.
+
+### Foundation for Advanced Wallet Features and Enhanced User Experience
+One of the underlying rationales of this proposal is to provide a robust foundation for more advanced wallet features. The dynamic nature of this approach offers wallets the opportunity to innovate. For example allowing users to support multiple dReps from a single account, or implementing sophisticated heuristics to maintain stake distribution to different pools. .
+
+## Specification
+
+### Overview
+
+Under this proposal, the constraints of [CIP-0011 ](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0011)regarding the sole derivation index for the stake key are removed. While we introduce the ability to derive stake keys beyond the first , there is no strict requirement to do so in a sequential manner, although it is recommended for logical organization. A pivotal change in this approach is that all derived stake keys must be associated with the first payment key. This consistent association to the first payment key serves as the foundation for the efficient discovery of these stake keys by wallet software and services.
+
+### Multi-Delegating
+- **Stake Key Derivation:** Begin by deriving the required number of stake keys needed for multi-delegation. While this derivation does not have to be sequential, it is recommended for clarity and orderly management. Ensure that the gap between any two consecutive active stake addresses does not exceed 20, in line with the [BIP-44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) address gap limit. 
+
+- **Key Registration:** Once derived, register these stake keys. This registration process remains consistent with current procedures, with the added context that all these new stake keys will be associated with a common payment key.
+
+- **Fund Distribution:** Distribute the funds across addresses formed by pairing the first  payment key with each of the newly created stake keys. This creates an association between all stake delegations. The details of how funds are distributed are specified by the user when creating the multi delegation transaction.
+
+Wallet software can then easily discover all stake keys and associated delegations either by sequential discovery or identifying addresses  containing the first payment credential. 
```

Comment:
> identifying addresses containing the first payment credential

This method of identification could incorrectly identify some stake credentials as belonging to the account of the first payment credential. Bob could create a mangled address with his own stake credentials and Alice's first payment credential. Bob could use this to make it appear that Alice has chosen to delegate some stake to a pool. 

Of course Alice, controlling the payment credential, could move control of the delegation to a different stake credential, but perhaps Alice isn't aware that this has happened or is inactive but staking her held ADA. Wallet software should only discover derived stake keys through discovery.

I didn't call it sequential discovery because according to this standard, the derivation does not need to be sequential, so sequential discovery could result in a subset of all derived keys.

Reply:
This needs to be reworded. The two methods of discovery are suggested here, sequential discovery is only possible to do for wallets. But external entities such as blockchain explorers or some dApps do not have access to the keys to do address derivation, but they can do the search by key association.

So wallets could either do sequential discovery (or association by key but then will need to take the extra precaution of pruning addresses which contains credentials it doesn't control).

External entities have to rely on key association (same as they do now for multi address wallets).

So this could probably be reworded as:

```
Wallet software can then easily discover all stake keys and associated delegations either by sequential discovery.
External entities such as blockchain explorers or dApps can do the discovery by identifying addresses containing the first payment credential.
```

thoughts?

Reply:
This is much better, just a small typo in the text itself.

> Wallet software can then easily discover all stake keys and associated delegations either by
sequential discovery.

Just remove the `either` and the sentence makes sense.